### ISSUE TITLE
Update to k8s 0.18.8

### ISSUE THREAD

**Issue by vaikas** [apache/camel-k#1698]
As part of the work going on in Knative and moving it to 0.18.8, I'm working on bringing eventing-contrib and as part of that work, looks like we need to bump the versions in this repo as well:

When I update the repo, I unsurprisingly get these kinds of errors:
```
vaikas-a01:eventing-contrib vaikas$ go test ./camel/...
# github.com/apache/camel-k/pkg/client/camel/clientset/versioned/typed/camel/v1
vendor/github.com/apache/camel-k/pkg/client/camel/clientset/versioned/typed/camel/v1/build.go:75:5: not enough arguments in call to c.client.Get().Namespace(c.ns).Resource("builds").Name(name).VersionedParams(&options, scheme.ParameterCodec).Do
	have ()
	want (context.Context)
vendor/github.com/apache/camel-k/pkg/client/camel/clientset/versioned/typed/camel/v1/build.go:92:5: not enough arguments in call to c.client.Get().Namespace(c.ns).Resource("builds").VersionedParams(&opts, scheme.ParameterCodec).Timeout(timeout).Do
	have ()
	want (context.Context)
vendor/github.com/apache/camel-k/pkg/client/camel/clientset/versioned/typed/camel/v1/build.go:109:8: not enough arguments in call to c.client.Get().Namespace(c.ns).Resource("builds").VersionedParams(&opts, scheme.ParameterCodec).Timeout(timeout).Watch
	have ()
	want (context.Context)
```

So, before this gets updated we won't be able to move the knative.dev/eventing-contrib forward. @nicolaferraro thoughts? You can see the kinds of changes required (and how to stage them) here:
https://github.com/knative/eventing-contrib/pull/1555

**Comment by nicolaferraro** [comment]
For reference, CamelSources are being moved into their own repo under https://github.com/knative-sandbox/eventing-camel.
Adding this to backlog, since an upgrade is needed nonetheless.

**Comment by astefanutti** [comment]
This has been fixed with #1753.
